deploy/kiali/kiali_cr.yaml

```diff
@@ -568,6 +592,9 @@ spec:
 #        type: "none"
 #        use_kiali_token: false
 #        username: ""
+#      component_status:
+#        app_label: ""
```

Comment:
values commented here should be exactly what the true default is for documentation purposes.
```suggestion
#        app_label: "jaeger"
```